packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -299,9 +299,117 @@ describe('FLE tests', () => {
     expect(compactResult).to.include('The "compactStructuredEncryptionData" command requires Mongo instance configured with auto encryption.');
   });
 
-  context('6.0+', () => {
-    skipIfServerVersion(testServer, '< 6.0'); // Queryable Encryption only available on 6.0+
-    skipIfServerVersion(testServer, '> 6.x'); // TODO(MONGOSH-1410): Queryable Encryption made a breaking change for 7.0
+  context('>= 6.0 && < 7.0', () => {
+    skipIfServerVersion(testServer, '< 6.0');
+    skipIfServerVersion(testServer, '>= 7.0');
```

Comment:
Does this fail on 7.x servers? (I can check later if you prefer)

Reply:
It seems to pass. I commented that line, focused the test and ran like this:

```
~/mongo/mongosh % MONGOSH_SERVER_TEST_VERSION=7.0.0-rc0  npx lerna run test --scope @mongosh/cli-repl
lerna notice cli v4.0.0
notice filter including "@mongosh/cli-repl"
info filter [ '@mongosh/cli-repl' ]
lerna info Executing command in 1 package: "npm run test"
lerna info run Ran npm script 'test' in '@mongosh/cli-repl' in 33.0s:

> @mongosh/cli-repl@0.0.0-dev.0 pretest
> npm run compile-ts


> @mongosh/cli-repl@0.0.0-dev.0 compile-ts
> tsc -p tsconfig.json


> @mongosh/cli-repl@0.0.0-dev.0 test
> cross-env TS_NODE_PROJECT=../../config/tsconfig.test.json mocha -r "../../scripts/import-expansions.js" --timeout 60000 -r ts-node/register "./{src,test}/**/*.spec.ts"



  FLE tests
global mongod is v6.0.5, wanted 7.0.0-rc0, downloading...
Downloading... https://downloads.mongodb.com/osx/mongodb-macos-x86_64-enterprise-7.0.0-rc0.tgz
Detected mongod version: 7.0.0
launching: "/Users/leroux.bodenstein/mongo/mongosh/tmp/mongodb-darwin-none-x64-700-rc0-enterprise/bin/mongod" on port 30120
replica set 'replset' initialized.
mongosh: downloading latest crypt shared library for inclusion in package: {"enterprise":true,"crypt_shared":true}
Skipping download because /Users/leroux.bodenstein/mongo/mongosh/tmp/crypt-store/host/mongodb-darwin-none-x64-continuous-enterprise exists
mongosh: downloaded /Users/leroux.bodenstein/mongo/mongosh/tmp/crypt-store/host/mongodb-darwin-none-x64-continuous-enterprise/lib/mongo_crypt_v1.dylib
    >= 6.0 && < 7.0
      ✔ can read QE data stored in a mongodb 6 database (2361ms)
Detected mongod version: 6.0.5
sent signal Signals.SIGTERM to 1 process.

```

 But I have no idea where `Detected mongod version: 6.0.5` comes from and if that matters..

Reply:
Do you think I should run this test on 7.x onwards as well?

Reply:
I removed that line and renamed the context() and it(). Let me know what you think.